### PR TITLE
Updated description, and added code to help stop the delay of carouse…

### PR DESCRIPTION
**Pull request by katharineking**
and added code to help stop the delay of the start of carousel.

### CODE REVIEW

File: about-us/index.html [hunk 1]
```diff
@@ -42,29 +42,28 @@
...
             <br>
-            We have organized a Hack Day this past November, and look forward to organizing another Hack Day this year
-            where we will spend the day enhancing our coding skills.
-
             <br>
+            We have also put a main focus on encouraging elementary and junior high students into the Computer Science field and
+            have been using the website
+            <i><a href="https://hourofcode.com/ca/learn" class="black-link" target="black">Hour of Code</a></i>
+            to help kids gain the skills to already start programming on their own!
+            We have already visited Windsor Park Elementary School and Carman Collegiate and all other
```
Comment (stoykoj) [inline about-us/index.html]: I believe it is called Windsor School. Looks awesome 😄

Comment (DiljotSG) [inline about-us/index.html]: "...and all other schools are welcome us to contact us by sending us an email at..."
This sentence does not make sense.

Consider rephrasing to: "...and other schools are welcome to contact us by email at..."
Other than that, it looks great (provided the school names are correct as Julia pointed out).
👍

Comment (katharineking) [reply about-us/index.html]: Windsor Elementary School! Just checked

Comment (katharineking) [reply about-us/index.html]: Thanks! Just changed it

**Pull request merged**